main.go

```diff
@@ -1,19 +1,27 @@
 package main
 
 import (
+	"encoding/json"
 	"net/http"
 	"sort"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/jmoiron/jsonq"
 	"github.com/resin-io/edge-node-manager/api"
 	"github.com/resin-io/edge-node-manager/application"
 	"github.com/resin-io/edge-node-manager/config"
 	"github.com/resin-io/edge-node-manager/process"
 	"github.com/resin-io/edge-node-manager/supervisor"
 )
 
+var version string
```

Comment:
Might be worth adding a comment here to say that this will be populated by the linker.